### ISSUE TITLE
Support for different embedded templates (like `ejs`)

### ISSUE THREAD

**Issue by marcelmokos** [amwmedia/node-plop#101]
Can I work on a PR that would add an option to change templating language from `handlebars` to `ejs` possibly other templating languages?

I am a user of `plop` ❤️ on many projects but I feel limited by `handlebars` templates and helpers. I would prefer to use Javascript as `ejs` allows. 
- https://github.com/mde/ejs

I do not want to create a similar library with different templating language when I like and use `plop` on my projects.

**Comment by amwmedia** [comment]
Yes, I'm very much in favor of this sort of ability so long as handlebars stays as the "default" so users aren't required to pick a template language, but they can override if they'd like.

**Comment by youssefsharief** [comment]
Any news about this?

**Comment by marcelmokos** [comment]
Unfortunately, I have decided to use https://www.hygen.io/ generator that is using `ejs` by default. I am not planning to work on this issue.

**Comment by youssefsharief** [comment]
@marcelmokos I also started using hygen. Thank you

**Comment by crutchcorn** [comment]
Transfering this issue to https://github.com/plopjs/plop/ since we're migrating all of `node-plop` to that repo as a monorepo
